must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -77,9 +77,9 @@ for resource in ${ceph_resources[@]}; do
 done
 
 # Inspecting the namespace where ceph-cluster is installed
-for ns in $(oc get cephcluster --all-namespaces --no-headers | awk '{print $1}'); do
+for ns in $(oc get csv --all-namespaces --no-headers |  awk '{print $1}'); do
```

Comment:
I don't think it will work as collecting CSV's namespaces can lead to an incorrect namespace list.